### DISCUSSION TITLE
M1 support

### DISCUSSION THREAD

**Discussion by egor-ryashin** [dirkschreib/SilentStudio#9] [Q&A] [answered]
I wonder if it was tested on M1 machines?

**Comment by dirkschreib** [comment, marked as answer]
> I wonder if it was tested on M1 machines?

You mean M1 without "Pro"/"Max"/"Ultra"?
AFAIK there is no Mac Studio in this configuration.

The best tested configuration is a Mac Studio with M1 Max.
The M2 Studios may have a different assignment of temperature sensors.
